### DISCUSSION TITLE
Project发展走向计划

### DISCUSSION THREAD

**Discussion by Minemetero** [lingbopro/easy-github-hosts#12] [Ideas]
ababababa

**Comment by Minemetero** [comment]
我们下一步实现什么功能呢？

**Comment by Minemetero** [comment]
我觉得可以加一个梯子的功能从本质上解决连接Github不顺畅的问题

**Comment by lingbopro** [comment]
> 我们下一步实现什么功能呢？

不到啊

**Comment by lingbopro** [comment]
> 我觉得可以加一个梯子的功能从本质上解决连接Github不顺畅的问题

那得要服务器罢  
而且这个项目本身就是基于HOSTS的

**Reply by Minemetero** [reply]
你说得很对但是Hosts未免太简单了,你来提个方案吧

**Reply by lingbopro** [reply]
> 你说得很对但是Hosts未免太简单了,你来提个方案吧

emmm，首先排除梯子或代理

**Comment by Minemetero** [comment]
我觉得可以加一个Hosts文件回档的功能

**Reply by lingbopro** [reply]
嗯，那得把原来的文件存起来  
而且这样的话是不是就得加一个功能  
或者让用户手动改也行

**Comment by Minemetero** [comment]
接下来该干些啥

**Reply by Minemetero** [reply]
我可以摆烂，你不可以。你修修我留下来的史山代码然后再加加功能，又可以水几个版本号。。。

**Reply by Minemetero** [reply]
不然这个repo你留下来的代码就不多了。。。

**Reply by Minemetero** [reply]
吸气，吐气。。。然后，倒头就睡，有缘再会。。。

**Reply by lingbopro** [reply]
> 我可以摆烂，你不可以。你修修我留下来的史山代码然后再加加功能，又可以水几个版本号。。。

呃啊

**Reply by lingbopro** [reply]
> 不然这个repo你留下来的代码就不多了。。。

我正好想到一个功能

**Reply by Minemetero** [reply]
啥功能？

**Reply by lingbopro** [reply]
> 啥功能？

回头做完了我push进`dev`分支

**Reply by Minemetero** [reply]
你快点做，我迫不及待了 :)

**Reply by lingbopro** [reply]
> 你快点做，我迫不及待了 :)

这周末做

**Comment by lingbopro** [comment]
我`双`出了个repo  
功能明后天做

**Reply by Minemetero** [reply]
:( 叫你同学干活

**Comment by lingbopro** [comment]
这个repo...

**Reply by Minemetero** [reply]
咋了

**Reply by lingbopro** [reply]
好久没更新了啊

**Reply by Minemetero** [reply]
稳定下来了还更新啥，你倒腾你的CSS我来更新，明天应该可以发
